src/api/events.js

```diff
@@ -0,0 +1,16 @@
+import Http from '../http';
+import Storage from '../storage';
+import Device from '../device';
+
+class Events {
+  static async sendEvent(eventData={}) {
+    const { type, metadata } = eventData;
+
+    const deviceId = Device.getId();
+    const userId = Storage.getItem(Storage.USER_ID);
+
+    return Http.request('POST', 'events', { type, metadata, deviceId, userId })
```

Comment:
Do we need to support `createdAt`? https://radar.com/documentation/api#send-a-custom-event

Reply:
yeah - this would be valuable if the sdk gets used server side. let me add before a release is cut.